src/main/java/ru/yandex/practicum/filmorate/model/ReleaseDateValidator.java

```diff
@@ -0,0 +1,16 @@
+package ru.yandex.practicum.filmorate.model;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.time.LocalDate;
+import java.time.Month;
+
+public class ReleaseDateValidator implements ConstraintValidator<ReleaseDate, LocalDate> {
+    @Override
+    public boolean isValid(LocalDate value, ConstraintValidatorContext context) {
+        if (value.isBefore(LocalDate.of(1895, Month.DECEMBER, 28))) {
```

Comment:
Можно сократить проверку до одной строчки:
```suggestion
        return !value.isBefore(LocalDate.of(1895, Month.DECEMBER, 28));
```